file/validate.go

```diff
@@ -29,5 +31,86 @@ func validate(content []byte) error {
 		err := fmt.Errorf(desc.String())
 		errs.Errors = append(errs.Errors, err)
 	}
+
+	var kongdefaults *KongDefaults
+	if err := json.Unmarshal(content, kongdefaults); err != nil {
```

Comment:
Have you actually tried running decK with this patch?
This won't work at all. The content in question should represents the following struct: https://github.com/Kong/deck/blob/main/file/types.go#L597-L617

Reply:
Yesterday's offline discussion KIC is a deck client, and the File.Content is the main construct I explored atm (edited) 
white_check_mark
eyes
raised_hands





10:04
Assuming decK applies the same interface to all clients.